ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -296,6 +296,9 @@ virtualization_type = args.virtType
 if virtualization_type == 'para':
     virtualization_type = 'paravirtual'
 
+if virtualization_type == 'hvm' and not sriov_type:
+    sriov_type = 'simple'
```

Comment:
Nope, this is not the solution to this problem. SRIOV requires a kernel module. While we have this module there is no guarantee that someone building their own image includes the module. Thus an image that gets this set automatically but does not actually have the module will have no network as the framework uses SRIOV enable HW instead of non SRIOV enabled HW.

The solution is a change in the way the image registration command is issued.